encoders/image/VSEImageEncoder/manifest.yml

```diff
@@ -2,13 +2,13 @@ manifest_version: 1
 name: VSEImageEncoder
 kind: encoder
 description: |
-    `VSEImageEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a
-        ndarray of `B x D`. Taken from the paper implementation of (VSE++: Improving Visual-Semantic Embeddings with Hard Negatives)
+    `VSEImageEncoder` encodes data from a ndarray, potentially BatchSize x (Channel x Height x Width) into a
+        ndarray of `BatchSize x D`. Taken from the paper implementation of (VSE++: Improving Visual-Semantic Embeddings with Hard Negatives)
     at https://github.com/fartashf/vseppauthor: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.4
+version: 0.0.5
 license: apache-2.0
 keywords: [Cross-Modal, Computer Vision, VSE, Hard Negative Mining, vsepp]
```

Comment:
I think we need to include one of the types: audio, image, nlp, video.

Reply:
ah good to know, thanks @theUnkownName !